### PR TITLE
examining the altar of gods as a ghost shows you some info

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

removes an unused favor global var

examining the altar of gods as a ghost shows you the deity name, religion name, bible name, sect, sect favor, examining it as an admin ghost shows you all chaplains as well

## Why It's Good For The Game

quick way for ghosts to see what the religion is 

## Changelog
:cl:
qol: examining the altar of gods as ghost shows you some info about the religion
/:cl:

### CODE REVIEW

**Comment by SmArtKar** [comment]
You should allow non-admin ghosts to see the religion/chaplains/other stuff as well?

**Comment by coiax** [comment]
Yeah, why is this admin only?

**Comment by tralezab** [comment]
I agree with the above two posters also thank you fikou

File: code/modules/religion/religion_structures.dm [hunk 1]
```diff
@@ -40,6 +40,21 @@
...
+	if(!isobserver(user))
+		return ..()
+	. = list("<span class='notice'><i>You examine [src] closer, and note the following...</i></span>")
+	if(GLOB.religion)
+		. += list("<span class='notice'>Deity: [GLOB.deity].</span>")
+		. += list("<span class='notice'>Religion: [GLOB.religion].</span>")
+		. += list("<span class='notice'>Bible: [GLOB.bible_name].</span>")
+	if(GLOB.religious_sect)
+		. += list("<span class='notice'>Sect: [GLOB.religious_sect].</span>")
+		. += list("<span class='notice'>Favor: [GLOB.religious_sect.favor].</span>")
```
Comment (Mothblocks) [inline code/modules/religion/religion_structures.dm:53]: Field: Value display is sooo boring

**Pull request merged**